queries/python/textobjects.scm

```diff
@@ -34,28 +34,182 @@
 (call (_) @call.inner)
 
 ;; Parameters
-((parameters
-  ([(identifier)
-   (tuple)
-   (typed_parameter)
-   (default_parameter)
-   (typed_default_parameter)
-   (list_splat)
-   (dictionary_splat)] @parameter.inner)
-   . ","? @_end)
-  (#make-range! "parameter.outer" @parameter.inner @_end))
-
-((lambda_parameters
-  [(identifier)
-   (tuple)
-   (typed_parameter)
-   (default_parameter)
-   (typed_default_parameter)
-   (list_splat)
-   (dictionary_splat)] @parameter.inner
-   . ","? @_end)
-  (#make-range! "parameter.outer" @parameter.inner @_end))
+
+(
+  (
+  parameters
+    "," @_start .
+    [
+      (identifier)
+      (tuple)
+      (typed_parameter)
+      (default_parameter)
+      (typed_default_parameter)
+      (dictionary_splat_pattern)
+      (list_splat_pattern)
+    ] @parameter.inner
+  )
+  (#make-range! "parameter.outer" @_start @parameter.inner)
+)
+
+(
+  (
+  parameters
+    "(" @justforfun .
```

Comment:
```suggestion
    "(" .
```

We should have our CI catch these :D

Reply:
fixed, I think it is a leftover from using the playground to test